react/RelatedProducts.js

```diff
@@ -26,7 +27,10 @@ const RelatedProducts = ({
   productList,
   recommendation: cmsRecommendation,
 }) => {
-  const productId = path(['product', 'productId'], productQuery)
+  const context = useContext(ProductContext)
```

Comment:
please use useProduct...

```import useProduct from 'vtex.product-context/useProduct'
const context = useProduct()
```

😬

Reply:
man, I wish we could kill the export of `ProductContext`